src/rachanaGJun20/PlindromeString.java

```diff
@@ -0,0 +1,41 @@
+package rachanaGJun20;
+
+import java.util.Scanner;
+
+public class PlindromeString {
+	//method will check whether given string is Palindrome
+	boolean isStringPlindrome(String str) {
+		String revstr = "";
+		String givenString = str;
+
+		for (int i = str.length() - 1; i >= 0; i--) {
+			revstr = revstr + str.charAt(i);
+		}
+		if (givenString.equals(revstr))
+			return true;
+		else
+			return false;
+	}
+	
+	public static void main(String[] args) {
+		PlindromeString plindromestring = new PlindromeString();
+		Scanner scanner = new Scanner(System.in);
+		boolean flag;
+		System.out.println("Pease Enter a string to check:");
```

Comment:
Suggestion: to reduce line of code ,
you can write like this as well

Rough Code : 
String[] stringArray = { "radar", "technocredits" };
		for (int i=0;i<stringArray.length;i++) {
		String string=stringArray[i];
			if (isStringPlindrome(string) == true)   //here method calling statement  depends on 
                                                                                     //Static or no Static
				System.out.println("Given String " + string + " is palindrome");
			else
				System.out.println("Given String " + string + " is not palindrome");
		}

Reply:
one thing I can surely do like I am repeating scanning two times,for this I can write loop,so,few lines for code I can reduce.

Reply:
Thanks for your input Ajit !
I will make these chnages as suggested.if (isStringPlindrome(string) == true)
But as  always try to make generalized logic for every possible program so that in future that program will get run for any given string without making any change in code. so not defining hard coded array .

Reply:
@RachanaGJun20 you can hardcode values, your method logic should be independent of hardcode values. from main method if you pass hardcode value that is accepted. Now you comfortable with scanner class, so now no need to use it.